charts/fluentd/templates/clusterrole.yaml

```diff
@@ -15,6 +15,7 @@ rules:
       - get
       - list
       - watch
+  {{- if and .Values.podSecurityPolicy.enabled (semverCompare "<=1.25-0" .Capabilities.KubeVersion.GitVersion) }}
```

Comment:
Shouldn't this be <1.25 ?

https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/

Reply:
Is GitVersion a thing? https://helm.sh/docs/chart_template_guide/builtin_objects/

I shall try

Reply:
Yes it is. If you bootstrap a Helm Chart with `helm create`, you can see this in the first few lines in the ingress definition.

Reply:
> Shouldn't this be <1.25 ?
> 
> https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/

Good catch, you are right.

Reply:
Can you patch for the version and merge everything together?

Reply:
Test with:
```
helm template fluentd ../fluentd/ --kube-version 1.25.1 -s templates/podsecuritypolicy.yaml
```

Reply:
Looks good to me. For versions >= 1.25 the psp configuration is removed automatically.

Reply:
> Can you patch for the version and merge everything together?

This can also be done with GitHubs "merge PR" to "base" branch.